.claude/skills/fix-crowdin-pr/SKILL.md

```
@@ -88,9 +88,29 @@ Do **not** use `GET /api/v2/projects/527308/translations?stringId=...` for verif
+- Applied: cite the fix commit sha and note the Crowdin upload was verified.
+- Skipped: state the reason (e.g. matches the English source, contradicts the file's register).
+
+Reply to a review comment (comment ids come from the step 1 fetch):
+
+```
+gh api -X POST repos/lukevella/rallly/pulls/<PR>/comments/<comment-id>/replies -f body='...'
+```
+
+Resolving needs GraphQL — list thread ids, then resolve the ones still open (coderabbit auto-resolves some threads when it sees the fix commit; skip those):
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)